packages/container/lib/fonts.module.scss

```diff
@@ -0,0 +1,102 @@
+@import "~@kaizen/hosted-assets/index";
+
+// =============== TIEMPOS ===============
+@font-face {
+  font-family: "Tiempos Headline";
+  src: url(asset-url("fonts/tiempos/tiempos-headline-bold.woff2")),
+    url(asset-url("fonts/tiempos/tiempos-headline-bold.woff"));
+  font-weight: 800;
+}
+
+// =============== GREYCLIFF ===============
```

Comment:
Is it safe to remove Greycliff fonts now?

Or should we wait until it's entirely purged from our codebase

Reply:
Yeah, given everyone is on Zen this should be removed.